### ISSUE TITLE
The speed of saving models during multi machine and multi card training is very slow

### ISSUE THREAD

**Issue by JarvisFei** [mit-han-lab/fastcomposer#15]
Have you tried training this model on multiple machines?

If you have tried, is there anything special to pay attention to in terms of environment settings and parameters？

**Comment by Guangxuan-Xiao** [comment]
Yes, we have. It's important to ensure a good inter-node connection to prevent communication bottlenecks during training.
